### PR TITLE
🎨 Social Impact Footer section

### LINKED ISSUE #19: 🎨  Create About us page (We care) section
#  Create About us page (We care) section
---

#### Note: this issue is dependent on #14, please complete that issue first

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a section that describes our mission and investment as an organisation

### Description

As seen on the designs (link below), this section contains the following:

```
1. A title
2. (2) paragraphs side by side
3. (3) Columns with an icon, title and test vertically aligned
```

The care section needs to appear as shown below (link to icons below)

Screenshot 2023-02-02 at 19 09 14

Your assignment is to implement the component to appear as above. The columns below use reusable components from #14. Please take note of that.

Please take note of all the design features (including rounded corners)

#### Link to full design:
```
https://xd.adobe.com/view/7b72572b-b40a-4618-bfa6-5500ab28f13b-2e21/screen/63e63506-4ea3-4d02-af28-f7cf148b47d9

mobile 

https://xd.adobe.com/view/7b72572b-b40a-4618-bfa6-5500ab28f13b-2e21/screen/0efb593f-dc28-4e6d-86f3-098699455ad4
```

#### Icon Links

https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690301/elewa-group-website/Icons/PNG/coorperative_kzlzrg.png

https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690300/elewa-group-website/Icons/PNG/Holistic_id4kra.png

https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690301/elewa-group-website/Icons/PNG/ownership_yno4a2.png

### Technical Analysis

This component will be created in  libs/pages/elewa/about-us

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
5. The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
6. The code is isolated to the libs/page/home package, unless a justified reason is specified.
7. The component is aligned using CSS flex
8. The design is responsive for screen, tablet and phone
9. The image renders fast and has multiple srcsets

### LINKED ISSUE #39: 🎨  Social Impact Footer section
#  Social Impact Footer section
---

#### Note: this feature depends on a component created on #252, please complete this issue first. 

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a footer section on the social impact page

### Description

Just add the component on #15 to the bottom of the about page.

Screenshot 2023-02-02 at 20 14 42

Your assignment is to implement the component to appear as above.

Please take note of all the design features 

#### Link to full design:
```
https://xd.adobe.com/view/7b72572b-b40a-4618-bfa6-5500ab28f13b-2e21/screen/31438bb0-400a-4da8-a374-c0440535c37c

mobile 
-
```
### Technical Analysis

This component will be created in  libs/pages/elewa/social-impact

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
6. The component is clean and does not have more than 50 lines
7. The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
8. The code is isolated to the libs/page/home package, unless a justified reason is specified.
9. The component is aligned using CSS flex
10. The design is responsive for screen, tablet and phone
11. The image renders fast and has multiple srcsets

### PR DESCRIPTION
**Pull request by Michael-Otieno**
# Description

Worked alone.

This pull request solves issue #39, whereby we have created a responsive About us page (We care) section following the design's guidelines.

Fixes #39

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Link a video or screenshot for both mobile and desktop showing the updates you made

File: libs/pages/elewa/home/src/lib/features-elewa-home.module.ts [hunk 1]
```diff
@@ -7,12 +7,13 @@ import { HomeJobsSectionComponent } from './components/home-jobs-section/home-jo
 import { HomeHeroSectionComponent } from './components/home-hero-section/home-hero-section.component';
 
 import { HomePageComponent } from './pages/home-page/home-page.component';
-
+import {PagesElewaSocialImpactModule} from '@elewa-group/pages/elewa/social-impact';
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:10]: Order of imports

Comment (Michael-Otieno) [reply libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:10]: what's the issue? I don't understand

Comment (LemmyMwaura) [reply libs/pages/elewa/home/src/lib/features-elewa-home.module.ts:10]: fix the order of imports.

File: libs/pages/elewa/social-impact/src/lib/social-impact/social-impact.component.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<elewa-group-footer></elewa-group-footer>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/social-impact/src/lib/social-impact/social-impact.component.html:1]: footer should only be on the main page

**Review by IanOdhiambo9 — COMMENTED** [review]
Closing PR due to conflicts and issue on hold

**Comment by IanOdhiambo9** [comment]
Please check issue #252 for footer implementation

**Pull request closed without merging**